Sources/Internal/FrameProvider.swift

```diff
@@ -152,6 +163,16 @@ final class FrameProvider {
     return CGRect(origin: CGPoint(x: x, y: y), size: daySize)
   }
 
+  func frameOfMonthFooter(_ month: Month, inMonthWithOrigin monthOrigin: CGPoint) -> CGRect {
+    let x = monthOrigin.x
+    let y = monthOrigin.y +
+        monthHeaderHeight +
+        content.monthDayInsets.top +
+        (monthsLayout.pinDaysOfWeekToTop ? 0 : (daySize.height + content.verticalDayMargin)) +
+        (CGFloat(numberOfRows(in: month)) * (daySize.height + content.verticalDayMargin))
+    return CGRect(origin: CGPoint(x: x, y: y), size: CGSize(width: monthWidth, height: monthFooterHeight))
```

Comment:
This `y` calculation should include the `monthDayInsets.bottom`, for the same reason explained here https://github.com/airbnb/HorizonCalendar/pull/75/files#r531198063

Reply:
Updated to include `monthDayInsets.bottom`